ml_genn/ml_genn/compilers/event_prop_compiler.py

```diff
@@ -1326,9 +1325,9 @@ def _build_adjoint_system(self, model: AutoNeuronModel,
                 # that are combined with l_V loss types
                 # This is at the moment categorically excluded
                 drive = sympy.Symbol("drive_p" if output else "RevISyn")
-                # add l^- - l^+ jump for neurons with A_reg regularisation
+                # add l^- - l^+ jump for neurons with regularisation
                 if regularise:
-                    drive += sympy.Symbol("drive_reg")
+                    drive += ((self.dt * self.example_timesteps)-sympy.Symbol("t"))*sympy.Symbol("drive_reg")
```

Comment:
should this have the same value as ``backT`` (``(self.dt * self.example_timesteps) - t - dt``)?

Reply:
oops ... backward time confusion is always possible ... it is similar, but it has to be non-zero, so I think an additional "-dt" is risky